app/src/actions/types.ts

```diff
@@ -19,25 +25,43 @@ interface ShowOrderInfoDrawer {
 
 interface SaveOrderbookTimestampInfo {
     type: typeof SAVE_ORDERBOOK_TIMESTAMP_INFO,
-    payload: SelectedTimestampInfo
+    payload: GlobalSelectedTimestampInfo
 }
 
-interface setDisableTransitions {
+interface SetDisableTransitions {
     type: typeof SET_DISABLE_TRANSITIONS,
     payload: boolean,
 }
 
+interface SetPlayback {
+    type: typeof SET_PLAYBACK,
+    payload: boolean,
+}
+
+interface SaveInstruments {
+    type: typeof SAVE_INSTRUMENTS,
+    payload: Array<string>,
+}
+
 // To expose generic actions:
 // export type GenericActions = FirstAction | SecondAction | ThirdAction
 
-export type GeneralActions = SaveReactAppName | ShowOrderInfoDrawer | SaveOrderbookTimestampInfo
-    | setDisableTransitions;
+export type GeneralActions = SaveReactAppName
+    | ShowOrderInfoDrawer
+    | SaveOrderbookTimestampInfo
+    | SetDisableTransitions
+    | SetPlayback
+    | SaveInstruments;
 
 export interface GeneralState {
     appName: string,
     showOrderInfoDrawer: boolean,
     orderDetails?: OrderDetails
     currentOrderbookTimestamp: string
     lastModificationType?: LastModificationType,
+    timeString?: string,
+    selectedDateNano: bigInt.BigInteger,
```

Comment:
What made you decide to make these optional? Wouldn't currentOrderBookTimeStamp and selectedDateNano be valid optional variables too?

Reply:
hmmm fair point, I don't think `timeString` should be optional here.

Reply:
by that same logic don't think lastModificationType should be optional either.